### PR TITLE
Added xcube-sh and xcube-cci

### PR DESCRIPTION
**Pull request by dzelge**
This PR will add the xcube plugins xcube-sh and xcube-cci to conda-forge

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/xcube-sh/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install requests_oauthlib -vv && {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
```
Comment (pont-us) [inline recipes/xcube-sh/meta.yaml:21]: There are some differences here with the dependencies list in environment.yml (e.g. here psycopg2 is missing).

**Review by pont-us — CHANGES_REQUESTED** [review]

**Comment by dzelge** [comment]
I agree to be a reviewer (dzelge)

**Comment by pont-us** [comment]
I agree to be a reviewer (pont-us)

File: recipes/xcube-cci/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - xcube >=0.5.0
+    - pip
```
[the remark this answers is not in the document]
Comment (scopatz) [reply recipes/xcube-cci/meta.yaml:23]: Is this? It isn't usually needed

**Comment by scopatz** [comment]
@pont-us are you willing to be a maintainer?

**Comment by pont-us** [comment]
@scopatz Yes, I am.

**Comment by dzelge** [comment]
@conda-forge/staged-recipes

Can you please merge or comment?

Thanx heaps

**Pull request merged**